qa/L0_backend_python/env/test.sh

```diff
@@ -63,6 +63,26 @@ cp ../../python_models/python_version/model.py ./models/python_3_7/1/
 cp python_backend/builddir/triton_python_backend_stub ./models/python_3_7
 conda deactivate
 
+# Use python-3-7 without conda pack
+path_to_conda_pack="$PWD/python-3-7"
+create_conda_env_with_specified_path "3.7" $path_to_conda_pack 
+conda install numpy=1.20.1 -y
+conda install tensorflow=2.1.0 -y
+conda install -c conda-forge libstdcxx-ng=12 -y
+
+PY37_VERSION_STRING="Python version is 3.7, NumPy version is 1.20.1, and Tensorflow version is 2.1.0"
```

Comment:
I'm not super familiar with this test, but it looks like this is redefining the previous PY37_VERSION_STRING with the same value before it's tested below. Is that intentional? Also, does this test fail before your changes in `python_backend`?

Reply:
If the test failed, the server would not have been able to run at all. I'll change the version string